src/js/components/FilterButtons.js

```diff
@@ -36,8 +36,8 @@ class FilterButtons extends React.Component {
     let filterCount = this.getCount(itemList);
 
     return filters.map((filter) => {
-      let classSet = classNames({
-        'button button-stroke button-inverse': true,
+      let classSet = classNames('button button-stroke',{
```

Comment:
I like the idea, but let's please stick to scope of this PR and extend the `FilterButtons` component in a different PR.